tests/openwisp2/settings.py

```diff
@@ -192,6 +192,7 @@
     OPENWISP_MONITORING_MAC_VENDOR_DETECTION = False
     OPENWISP_MONITORING_API_URLCONF = 'openwisp_monitoring.urls'
     OPENWISP_MONITORING_API_BASEURL = 'http://testserver'
+    OPENWISP_MONITORING_AUTO_SNMP_DEVICEMONITORING = True
```

Comment:
please avoid this and create the check manually in the tests so we avoid having to change all the other tests (which is not good).